build/x86_64_sles12/Dockerfile

```diff
@@ -28,9 +28,9 @@ RUN zypper addrepo https://download.opensuse.org/repositories/devel:libraries:c_
         varnish-devel \
         wget \
         which \
- && wget ftp://ftp.pbone.net/mirror/ftp5.gwdg.de/pub/opensuse/repositories/home:/Herbster0815/openSUSE_Factory/noarch/automake-1.14.1-56.1.noarch.rpm \
- && rpm --force -Uvh automake-1.14.1-56.1.noarch.rpm \
- && rm automake-1.14.1-56.1.noarch.rpm \
- && zypper -n clean \
+ && wget https://ftp.gwdg.de/pub/opensuse/repositories/home%3A/Herbster0815/openSUSE_Leap_15.1/noarch/automake-1.16.1-lp151.92.1.noarch.rpm \
+ && rpm --force -Uvh automake-1.16.1-lp151.92.1.noarch.rpm \
+ && rm automake-1.16.1-lp151.92.1.noarch.rpm \
```

Comment:
Careful — I think our current build requires automake 1.14 (automake versions aren't backward-compatible). Did you test the build steps in the new container?

Reply:
Actually, our current implementation requires automake 1.14 or later. The one that comes with OpenSUSE Leap 42.3 is apparently too old.